### PR TITLE
Backend/feat:optional-fare-recompute-and-show

### PR DESCRIPTION
**Pull request by enigma-commits**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs [hunk 1]
```diff
@@ -908,10 +908,6 @@ getConfigValue language issueConfig mbRideInfoRes key = do
         "ESTIMATED_FARE" -> show estimatedFare
         "FINAL_FARE" -> show finalFare
         "FARE_DIFFERENCE" -> show fareDifference
```
Comment (hkmangla) [inline Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs:910]: why did you remove this?

**Review by hkmangla — CHANGES_REQUESTED** [review]

File: Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs (hunk 1 above)
Comment (enigma-commits) [reply Backend/lib/shared-services/src/IssueManagement/Domain/Action/UI/Issue.hs:910]: oh sorry i accidently committed this causes issue in running dsl because of using rupee sign.

**Review by hkmangla — APPROVED** [review]
